### ISSUE TITLE
[FEATURE] % BoS

### ISSUE THREAD

**Issue by skvie** [becurrie/titandash#483]
**Is your feature request related to a problem? Please describe.**
Currently the bot just add relics into Artifacts defined.

**Describe the solution you'd like**
Have an additional parameter to define how long a certain artifact should be updated (in % of lifetime) 
**Describe alternatives you've considered**
-
**Additional context**
-

**Comment by skvie** [comment]
I'll look into this

**Comment by becurrie** [comment]
@breezko has been working on getting something in place to make this possible. I'm leaving this issue here until we're ready to begin implementing it.

Will likely need some backing issues resolved before this can be done (save file reader implementation within bot).

**Comment by breezko** [comment]
@becurrie Yeah, the current implementation has a webservice to decode the gamefile, which will be serialized to a python object. 
I'll do later some benchmarks on how fast the IO is and could possible be "available" for the bot to access.
